pkg/alertmanager/amcfg.go

```diff
@@ -2020,6 +2066,18 @@ func (whc *webhookConfig) sanitize(amVersion semver.Version, logger log.Logger)
 	return nil
 }
 
+func (tc *msTeamsConfig) sanitize(amVersion semver.Version, logger log.Logger) error {
+	if amVersion.LT(semver.MustParse("0.26.0")) {
+		return fmt.Errorf(`invalid syntax in receivers config; msteams integration is only available in Alertmanager >= 0.25.1`)
+	}
```

Comment:
Is it >= 0.26.0 or >= 0.25.1?

Reply:
MS Teams integration came with Alertmanager 0.26.0

Reply:
We probably want to fix the logline then :)

Reply:
Good catch 😄 !
Fixed thanks folks